### PR TITLE
Update A9 - Cimentech README

### LINKED ISSUE #120: Add install Ruby instructions into A9 ATTACK.md
Some developers may not have installed it already in their machines.

### LINKED ISSUE #212: A9 Cimentech not exploitable on Linux machines
After building the app, it is not possible to follow the attack narrative and reproduce the exploitation.

Maybe docker running in Linux has some different protections.

Issue found by @mbenford . Thanks for that! 👏🏻

### PR DESCRIPTION
**Pull request by Krlier**
This PR aims to update A9 - Cimentech's README to contemplate the attack narrative as well.

The `docs` folder has been removed and it's images have been moved to the `images` folder, at the app's root directory.

## Closes #212 
After investigating, I found out that, after installing Ruby on Linux systems, some might come without the Highline gem, which is needed to run the exploit properly. 

In order to fix this, I added a reference on `README.md` to issue #212, which has the steps on how to fix it.

## Closes #120
I added a reference to Ruby's installation [webpage][1] on `README.md`, which helped me get the job done.

[1]: https://www.ruby-lang.org/en/documentation/installation/

### CODE REVIEW

File: owasp-top10-2017-apps/a9/cimentech/README.md [hunk 1]
```diff
@@ -1,43 +1,132 @@
...
+
+```sh
+ruby /tmp/44449.rb http://localhost
+```
+
+<img src="images/attack5.png" align="center"/>
+
+**NOTE**: You need to have Ruby installed on your system to run the exploit, for information on how to install it, click [here][1]!
+
+**NOTE 2**: If you are running Linux and came across an execution error when trying to run the exploit, please have a look at this [Issue][4] for information on how to proceed.
```
Comment (Takehime) [inline owasp-top10-2017-apps/a9/cimentech/README.md]: I had the exact same issue running ruby on MacOS. I think this issue might happen in other systems as well. What do you think about just mentioning the issue, not necessarily Linux?

File: owasp-top10-2017-apps/a9/cimentech/README.md [hunk 2]
```diff
@@ -1,43 +1,132 @@
...
+- [Setup](#setup)
+- [Attack narrative](#attack-narrative)
+- [Objectives](#secure-this-app)
+- [Solutions](#pr-solutions)
+- [Contributing](#contributing)
 
-Definition from [OWASP](https://www.owasp.org/images/7/72/OWASP_Top_10-2017_%28en%29.pdf.pdf):
+## What does it mean to use a component with known vulnerabilities?
 
 Components, such as libraries, frameworks, and other software modules, run with the same privileges as the application. If a vulnerable component is exploited, such an attack can facilitate serious data loss or server takeover. Applications and APIs using components with known vulnerabilities may undermine application defenses and enable various attacks and impacts.
```
Comment (rafaveira3) [inline owasp-top10-2017-apps/a9/cimentech/README.md]: What do you think of adding the following at the beginning of this paragraph: `Imagine that components, such as ....`

File: owasp-top10-2017-apps/a9/cimentech/README.md (hunk 1 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a9/cimentech/README.md]: Thanks for your feedback @Takehime! I'll add that!

File: owasp-top10-2017-apps/a9/cimentech/README.md (hunk 2 above)
Comment (Krlier) [reply owasp-top10-2017-apps/a9/cimentech/README.md]: Thanks @rafaveira3! It does look better with what you proposed!

**Pull request merged**